packages/terra-compact-interactive-list/src/subcomponents/Row.jsx

```diff
@@ -161,7 +168,7 @@ const Row = (props) => {
       data-row-id={id}
       role={activeRow && 'row'}
       aria-hidden={activeRow ? null : true}
-      className={cx('row', isTopmost && 'row-topmost', isLeftmost && 'row-leftmost', !activeRow && 'row-placeholder', theme.className)}
+      className={cx('row', !hasVisibleBorders && 'borderless', isTopmost && 'row-topmost', isLeftmost && 'row-leftmost', !activeRow && 'row-placeholder', theme.className)}
```

Comment:
since there are so many classnames here now, for readability can we separate this out into a variable similar to how it's done in Slide Panel:

https://github.com/cerner/terra-framework/blob/main/packages/terra-slide-panel/src/SlidePanel.jsx#L155-L160

Reply:
refactored: https://github.com/cerner/terra-framework/pull/2075/commits/27d253d7d771c7c99c81e0a05162723569d6ed08